components/Gallery/GalleryItem.tsx

```diff
@@ -2,27 +2,22 @@ import Image from "next/image";
 import styles from "./GalleryItem.module.scss";
 
 interface Props {
-  readonly src: string;
+  readonly src: StaticImageData;
   readonly title: string;
 }
-/**
- * TODO:
- * in `a` tag use thumbnail image
- * in `Image` use high quality full image
- */
+
 export const GalleryItem = ({ src, title }: Props) => {
   return (
-    <li className={styles.item}>
-      <a href={src}>
+    <a className={styles.item} href={src.src}>
         <Image
           src={src}
           alt={title}
           width="100%"
           height="100%"
+          sizes="(max-width: 767px) 30vw, (max-width: 1200px) 25vw, 17vw"
```

Comment:
this is an interesting prop, thanks to that I was able to finally load correct size of image